### PR TITLE
Use only airtop sdk API

### PR DESCRIPTION
**Pull request by dreamerns**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -15,49 +13,39 @@ async function run() {
...
     const windowId = windowInfo.data.windowId;
 
     console.log('Summarizing content...');
-    const contentSummary = await client.windows.promptContent(session.id, windowId, { prompt: 'Summarize the content of the page in 1 paragraph' }); // Note that scrapeContent is also available to do a clean scrape of the page content
+    const contentSummary = await client.windows.promptContent(sessionId, windowId, { prompt: 'Summarize the content of the page in 1 paragraph' }); // Note that scrapeContent is also available to do a clean scrape of the page content
     console.log('Content summary:\n\n', chalk.green(contentSummary.data.modelResponse));
 
     // Clean up
-    await browser.close();
-    await client.sessions.terminate(session.id);
```
Comment (pciporin25) [inline index.ts:60]: @dreamerns should we call `client.windows.close` for cleanup here?

Comment (pciporin25) [reply index.ts:60]: Also, wondering if we should add a code comment to help users who might want to see the live view after the code execution is complete.  Something like
```
// Uncomment these lines if you wish to access your live view after the script has completed.
```

File: index.ts [hunk 2]
```diff
@@ -15,49 +13,39 @@ async function run() {
...
 
-    // Open a new page
-    const page: Page = await browser.newPage();
-    
-    // Navigate to the URL
-    console.log('Navigating to URL', TARGET_URL);
-    await page.goto(TARGET_URL);
-    const windowInfo = await client.windows.getWindowInfoForPuppeteerPage(session, page, {
+    const windowInfo = await client.windows.getWindowInfo(sessionId, windowResponse.data.windowId, {
       disableResize: true, // Prevents the browser window from being resized when loading a live view, which might impact the agent's ability to scrape or summarize content
```
Comment (pciporin25) [inline index.ts]: Did we decide to leave `disableResize`?  Or would we be better off removing it?

Comment (dreamerns) [reply index.ts]: I'm in favor of removing it

File: index.ts (hunk 1 above)
Comment (dreamerns) [reply index.ts:60]: @pciporin25 I guess we can just to show more of the API, but it doesn't really matter right? You don't have to close the window before closing the browser.

**Review by pciporin25 — APPROVED** [review]

**Pull request merged**